tests/test_python_cli.sh

```diff
@@ -10,24 +10,3 @@ set -ex
 keygenerator.sh --help
```

Comment:
It looks like we can get rid of this file too.

Reply:
Apart from `--help`, we already call it to create members/users' keys/certs in the CI so I think it's OK to delete this file.